### PR TITLE
Support 8 bit content transfer encoding

### PR DESCRIPTION
**Pull request by say-yawn**
# About this PR
Support 8-bit Content Transfer Encoding by using `message_from_bytes` instead of `message_from_string`.

# Acceptance Criteria
- [ ] Email sent with 8-bit Content Transfer Encoding(CTE) no longer has broken body message
- [ ] Email sent with 8-bit CTE is converted to base64
- [ ] Follow-up issue created to document that emails coming in may not have UTF-8 as Content-Type's charset which can cause breakage

### CODE REVIEW

File: emails/views.py [hunk 1]
```diff
@@ -299,8 +299,33 @@ def _sns_message(message_json):
...
+        bytes_email_message
+    )
+    strip_texts = []
+    for item in mail['headers']:
+        for k, v in item.items():
+            strip_texts.append(': '.join([k, v]))
+    stripped_content = message_json['content']
+    for item in strip_texts:
+        stipped_content = stripped_content.replace(item, '')
+    logger.error('body_message', extra={
```
Comment (groovecoder) [inline emails/views.py:317]: Still need this `logger.error` line?

File: emails/views.py [hunk 2]
```diff
@@ -310,7 +335,7 @@ def _sns_message(message_json):
     if html_content:
```
Comment (groovecoder) [inline emails/views.py:335]:
```suggestion
    if b_html_content:
```

File: emails/views.py [hunk 3]
```diff
@@ -299,8 +299,33 @@ def _sns_message(message_json):
     email_message = message_from_string(
         message_json['content'], policy=policy.default
     )
+    bytes_email_message = message_from_bytes(
+        message_json['content'].encode('utf-8'), policy=policy.default
+    )
 
     text_content, html_content, attachments = _get_all_contents(email_message)
```
Comment (groovecoder) [inline emails/views.py:306]: Can we remove this now if we're only using the `b_` variables?

**Review by groovecoder — CHANGES_REQUESTED** [review]

**Comment by groovecoder** [comment]
Code and spot-checks on dev look good.

**Review by groovecoder — APPROVED** [review]

**Comment by sv-hyacoub** [comment]
We followed the steps from [#655](https://github.com/mozilla/fx-private-relay/issues/655) to verify the first test case of this PR, but the body of the email is still broken in every email service provider and the header of the email is displayed as following:
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: base64

For the second test case we verified that special characters are not correctly displayed in email body section when sending emails from thunderbird to every email service provider via Firefox Relay. We confirm that the email body is displayed correctly now but the header of the email is displayed as following:
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: quoted-printable

@say-yawn could you please take a look at out testing results and let's know your thoughts?

**Pull request merged**